frontend/public/components/storage-class-form.tsx

```diff
@@ -64,105 +65,29 @@ export class StorageClassForm_ extends React.Component<
 
   storageTypes = {};
 
+  // Fetch Storage type provisoners from different operators
+  // For CSI - provionerType: 'csi'
+  // For Defaults - provionerType: 'others'
+  getExtensionsStorageClassProvisioners = (provionerType = 'others') => {
+    const extensionCSIProvisoners = {};
+    _.forEach(plugins.registry.getStorageClassProvisioner(), (operator: any) => {
```

Comment:
dont use registry, but HOC `withExtensions` to make the component reactive to extension changes